### ISSUE TITLE
GameVersion error

### ISSUE THREAD

**Issue by moth1995** [kitserver/kitserver5#3]
Hi! 

i notice that when kitserver5 its attached to WE9 it get recognize as PES5, i made test with many executables and its the same all the time, i notice this bug when i was using the network module because i was getting a 404 and then i made some test first to discard error at my server i made an curl request from another server and i was getting 200, then with wireshark filtering by http protocol, i get notice that the folder that was requestion was pes5 and not we9, then i opened the log and found this line

{KLOAD} Game version: PES5 PC

is this an important bug? or is nothing we should really care because they share the same memory address?

thanks!

**Comment by juce** [comment]
yeah, i think there are lots of we9.exe files on the internet, which are actually not real executables from Winning Eleven 9, but copies of a no-dvd version of pes5.exe. Since both games are 100% compatible in terms of data (but not compatible with WE9LE), you can use the exes interchangeably. Kitserver does not look at the filename, but seaches inside the exe for the information - and is therefore detecting it as "PES5 PC". I don't know if i saw ever saw a proper no-dvd we9.exe that was true we9 and not a copy of pes5 exe. (If you try it with original we9.exe from dvd disk, with securom protection and such, then kitserver should recognize it as "WE9 PC").

And that is why the network module does what it does.
Not sure we can call it a bug, but yeah, it's a bit confusing behaviour for sure :)
